Create organization README.md (#3)
graalvm/.github: profile/README.md

## GraalVM on GitHub

![GraalVM banner](https://raw.githubusercontent.com/graalvm/.github/main/graalvm-github.png "GraalVM on GitHub")

GraalVM is a high-performance JDK designed to accelerate Java application performance while consuming fewer resources. 

> It offers two ways to run Java applications: on the HotSpot JVM with Graal just-in-time compiler or as an ahead-of-time compiled native executable. Besides Java, it also provides runtimes for JavaScript, Ruby, Python, and a number of other popular languages with polyglot capabilities.

Reviewer: ```suggestion
It offers two ways to run a Java application: on the HotSpot JVM with the Graal just-in-time compiler or as an ahead-of-time compiled native executable. Besides Java, it also provides runtimes for JavaScript, Ruby, Python, and a several other popular languages with polyglot capabilities.
```

> ## Get started

Reviewer: ```suggestion
## Get Started
```

* [Website](https://www.graalvm.org/)
* [Documentation](https://www.graalvm.org/latest/docs/)
* [Downloads](https://www.graalvm.org/downloads/)
* [Docker images](https://github.com/orgs/graalvm/packages)
* [Demos](https://github.com/graalvm/graalvm-demos)

> ## GraalVM's GitHub repositories

Reviewer: ```suggestion
## GraalVM's GitHub Repositories
```

Name         | Description
------------ | -------------
[Graal](https://github.com/oracle/graal) | The core repository
[GraalVM CE Builds](https://github.com/graalvm/graalvm-ce-builds) | GraalVM CE builds
[GraalVM CE Dev Builds](https://github.com/oracle/graal) | GraalVM CE Developer builds
[Native Build Tools](https://github.com/graalvm/native-build-tools) | Build plugins for GraalVM Native Image

> [Graalvm Reachability Metadata](https://github.com/oracle/graalvm-reachability-metadata) | Reachability metadata for open-source libraries

Reviewer: ```suggestion
[GraalVM Reachability Metadata](https://github.com/oracle/graalvm-reachability-metadata) | Reachability metadata for open-source libraries
```

[FastR](https://github.com/oracle/fastr) | Implementation of the R language
[GraalJS](https://github.com/oracle/graaljs) | Implementation of JavaScript and Node.js
[GraalPy](https://github.com/oracle/graalpython) | Implementation of the Python language
[TruffleRuby](https://github.com/oracle/truffleruby) | Implementation of the Ruby language
[VS Code Extensions](https://github.com/graalvm/vscode-extensions) | GraalVM's extensions for VS Code
[GraalVM Demos](https://github.com/graalvm/graalvm-demos) | Several example applications illustrating GraalVM capabilities
[Simple Language](https://github.com/graalvm/simplelanguage) | A simple example language built with the Truffle framework
[Simple Tool](https://github.com/graalvm/simpletool) | A simple example tool built with the Truffle framework

## Contribute

We welcome community contributions! You can find contributing guidelines in each GraalVM repository.
A few additional resources:
* [Learn how to become a contributor](https://www.graalvm.org/community/contributors/)
* [Understanding how to build GraalVM from source](https://www.youtube.com/watch?v=3Gh0cz3vjG8)

> ## Get help

Reviewer: ```suggestion
## Get Help
```

* For bug reports and feature requests, please create a GitHub issue
* For general questions and discussions, start a GitHub discussion

> * Additionally, you can send an email to [graalvm-users@oss.oracle.com](mailto:graalvm-users@oss.oracle.com) or join conversation on [GraalVM's Slack](https://www.graalvm.org/slack-invitation/)

Reviewer: ```suggestion
* Additionally, you can send an email to [graalvm-users@oss.oracle.com](mailto:graalvm-users@oss.oracle.com) or join the conversation on [GraalVM's Slack](https://www.graalvm.org/slack-invitation/)
```

* To report security vulnerabilities, please send an email to [secalert_us@oracle.com](mailto:secalert_us@oracle.com) 

> ## Get in touch

Reviewer: ```suggestion
## Get in Touch
```

## GraalVM on GitHub

![GraalVM banner](https://raw.githubusercontent.com/graalvm/.github/main/graalvm-github.png "GraalVM on GitHub")

GraalVM is a high-performance JDK designed to accelerate Java application performance while consuming fewer resources. 
It provides the Graal compiler, which can be used as a just-in-time compiler to run Java applications on the HotSpot JVM or to ahead-of-time compile them into native executables. Besides Java, it also provides runtimes for JavaScript, Ruby, Python,  and a several other popular languages with polyglot capabilities.

## Get Started

* [Website](https://www.graalvm.org/)
* [Documentation](https://www.graalvm.org/latest/docs/)
* [Downloads](https://www.graalvm.org/downloads/)
* [Docker images](https://github.com/orgs/graalvm/packages)
* [Demos](https://github.com/graalvm/graalvm-demos)

## GraalVM's GitHub Repositories

Name         | Description
------------ | -------------
[Graal](https://github.com/oracle/graal) | The core repository
[GraalVM CE Builds](https://github.com/graalvm/graalvm-ce-builds) | GraalVM CE builds

> [GraalVM CE Dev Builds](https://github.com/oracle/graal) | GraalVM CE Developer builds

Reviewer 2: ```suggestion
[`graalvm/graalvm-ce-dev-builds`](https://github.com/graalvm/graalvm-ce-dev-builds) | GraalVM CE Developer builds
```

[Native Build Tools](https://github.com/graalvm/native-build-tools) | Build plugins for GraalVM Native Image
[GraalVM Reachability Metadata](https://github.com/oracle/graalvm-reachability-metadata) | Reachability metadata for open-source libraries
[Setup GraalVM](https://github.com/oracle/fastr) | GitHub Action for GraalVM
[GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader) | GraalVM JDK Downloader
[FastR](https://github.com/oracle/fastr) | Implementation of the R language
[GraalJS](https://github.com/oracle/graaljs) | Implementation of JavaScript and Node.js
[GraalPy](https://github.com/oracle/graalpython) | Implementation of the Python language
[TruffleRuby](https://github.com/oracle/truffleruby) | Implementation of the Ruby language
[GraalVM Demos](https://github.com/graalvm/graalvm-demos) | Several example applications illustrating GraalVM's capabilities
[Simple Language](https://github.com/graalvm/simplelanguage) | A simple example language built with the Truffle framework

> [Simple Tool](https://github.com/graalvm/simpletool) | A simple example tool built with the Truffle framework

Reviewer 2: I'd prefer the GitHub slug notation here, so instead of:
```
[Graal](https://github.com/oracle/graal) | The core repository
```
it'd be nice to have:
```
[`oracle/graal`](https://github.com/oracle/graal) | The GraalVM core repository
```

This will also avoid some weird duplication (e.g., `GraalVM CE Builds`).

## Contribute

We welcome community contributions! You can find contributing guidelines in each GraalVM repository.
A few additional resources:
* [Learn how to become a contributor](https://www.graalvm.org/community/contributors/)
* [Understanding how to build GraalVM from source](https://www.youtube.com/watch?v=3Gh0cz3vjG8)

## Get Help

> * For bug reports and feature requests, please create a GitHub issue

Reviewer 2: ```suggestion
* For bug reports and feature requests, please [create a GitHub issue](https://github.com/oracle/graal/issues/new/choose)
```

> * For general questions and discussions, start a GitHub discussion

Reviewer 2: ```suggestion
* For general questions and discussions, [start a GitHub discussion](https://github.com/oracle/graal/discussions/new/choose)
```

> * Additionally, you can send an email to [graalvm-users@oss.oracle.com](mailto:graalvm-users@oss.oracle.com) or join the conversation on [GraalVM's Slack](https://www.graalvm.org/slack-invitation/)

Reviewer 2: ```suggestion
* Additionally, you can join the conversation on [GraalVM's Slack](https://www.graalvm.org/slack-invitation/) or send an email to [graalvm-users@oss.oracle.com](mailto:graalvm-users@oss.oracle.com)
```
